### ISSUE TITLE
http-post-form using single tcp connection?

### ISSUE THREAD

**Issue by samhamra** [vanhauser-thc/thc-hydra#460]
Is it possible to somehow send multiple post requests using the same tcp connection, perhaps by using the Connection: Keep-Alive HTTP header?
I tried just adding the header to the command like so:

 hydra -O -v -V -I -l administrator -x 1:1:aA myip https-post-form "mypath:myxml:failtext:H=Content-Type: application/xml; Connection: Keep-Alive"

But it seems like I wasn't successful, is there another way?

**Comment by vanhauser-thc** [comment]
no, hydra only does connection: close to keep the protocol complexity as low as possible.
to do 1.1 keep-alive would mean we need a http library to take care of that.
of course that would be a good thing (it also increases performance) but we dont have the time for it at the moment.

but pull requests with at would be more than welcome!
